plant/plants/tests/test_views.py

```diff
@@ -0,0 +1,62 @@
+import pytest
+from django.urls import reverse
+from django.utils import timezone
+
+from plant.plants.tests.factories import PlantFactory, WateringFactory
+
+
+class TestPlantViewSet:
+    @pytest.mark.django_db()
+    def test_list_plants(self, client):
+        plants = PlantFactory.create_batch(size=5)
+        url = reverse("plants:plants-list")
+        response = client.get(url)
+        assert response.data[0]["name"] == plants[0].name
+        assert response.status_code == 200
+
+    @pytest.mark.django_db()
+    def test_get_plant(self, client):
+        plants = PlantFactory.create_batch(size=5)
+        url = reverse("plants:plants-detail", kwargs={"pk": plants[0].id})
+        response = client.get(url)
+        assert response.data["name"] == plants[0].name
+        assert response.status_code == 200
+
+    @pytest.mark.django_db()
+    def test_delete_plant(self, client):
+        plants = PlantFactory.create_batch(size=5)
+        url = reverse("plants:plants-detail", kwargs={"pk": plants[0].id})
+        url_plants_list = reverse("plants:plants-list")
+        response = client.delete(url)
+        plants_list = client.get(url_plants_list)
+        assert response.status_code == 204
+        assert len(plants_list.data) == 4
+
+    @pytest.mark.django_db()
+    def test_add_plant(self, client):
+        url = reverse("plants:plants-list")
+        payload = dict(
```

Comment:
It is not a mistake using dict and I also like it a lot, but more popular convention in our company is to stick with the typical dict creation
```
{
    "name": "TEST",
    "other_keys": "other_values"
}
```